index.html

```diff
@@ -1,16 +1,124 @@
+                        ext ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived 
+                        not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s 
+                        with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus 
+                        PageMaker including versions of Lorem Ipsum.
+                   </p>
+               </div>
+                <div class="col-sm-8">
+                   <img src="/images/david-holifield-kPxsqUGneXQ-unsplash.jpg" alt="image" width="700"> 
+
+                   <div class="row">
+                        <div class="col-sm-2">
+                            <img src="/images/mums-w-frame.jpg" alt="" width="100">
+                        </div>
+                        <div class="col-sm-2">
+                            <img src="/images/mums-w-frame.jpg" " alt="" width="100">
+                        </div>
+                        <div class="col-sm-2">
+                            <img src="/images/mums-w-frame.jpg"  alt="" width="100">
+                        </div>
+                        <div class="col-sm-2">
+                            <img src="/images/mums-w-frame.jpg" " alt="" width="100">
+                        </div>
+                    </div>
+
+                </div>
+
+           </div>
+           
+           <div class="row">
+                <div class="col-sm-3">
+                    <img src="/images/mums-w-frame.jpg" alt="" width="100">
+                    <figcaption>Some info</figcaption>
+                </div>
+                <div class="col-sm-3">
+                    <img src="/images/mums-w-frame.jpg" " alt="" width="100">
+                    <figcaption>Some info</figcaption>
+                </div>
+                <div class="col-sm-3">
+                    <img src="/images/mums-w-frame.jpg"  alt="" width="100">
+                    <figcaption>Some info</figcaption>
+                </div>
+                <div class="col-sm-3">
+                    <img src="/images/mums-w-frame.jpg" " alt="" width="100">
+                    <figcaption>Some info</figcaption>
+                </div>
+            </div>
+       </main>
+       <footer class="card-footer bg-info">
+           <ul class="footer-list">
+               <li>
+                    <a id="footer-links" href="#">Site Map</a>
+                </li>
+               <li>
+                    <a id="footer-links" href="#">Delivery</a>
+                </li>
+               <li>
+                    <a id="footer-links" href="#">Contact</a>
+                </li>
+           </ul>
+       </footer>
+
+
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" 
```

Comment:
CDN declarations should go in the head

Reply:
I could be very wrong on this but we were told to add script just under the body tag? Is that info wrong?

Reply:
CDNs and libraries should be included in the head as they are classed as dependencies. i.e code below them in the DOM relies on them to work. You should generally put functions and code that directly affects the DOM in the body at the end of the last child element. This is because if the code requires DOM element to reference (which it almost certainly will) it will need to wait for the DOM elements in question to be loaded first and made accessible. i.e if the script is at the start the elements wont be accessible to manipulate yet.

A quick way to think of it is pretty much:

head - libraries, CDNs, functions which need to be called on page load (e.g google analytics scripts)
body - everything else that depends on interacting with the DOM elements

Reply:
So yes you were taught very much correctly, there are just few exceptions to the rule to be aware of